src/test/scala/org/apache/spark/sql/delta/DeltaSourceSuite.scala

```diff
@@ -53,8 +53,7 @@ class DeltaSourceSuite extends DeltaSourceSuiteBase {
       new File(inputDir, "_delta_log").mkdir()
       val e = intercept[AnalysisException] {
         spark.readStream
-          .format("delta")
-          .load(inputDir.getCanonicalPath)
+          .delta(inputDir.getCanonicalPath)
```

Comment:
I don't know whether we need to refactor all the tests here. It seems better to just have a single new one to exercise each of the new implicits.

Reply:
Thanks a lot for your feedback @jose-torres 🙏, I made the requested changes.